test/views/errors/SessionTimeoutViewSpec.scala

```diff
@@ -29,30 +28,21 @@ class SessionTimeoutViewSpec extends ViewBaseSpec {
     object Selectors {
       val pageHeading = "#content h1"
       val instructions = "#content p"
-      val link = "#content p a"
     }
 
-    lazy val mockAppConfig: MockAppConfig = new MockAppConfig {
-      override val ggServiceUrl: String = "sign-in"
-    }
-
-    lazy val view = views.html.errors.sessionTimeout(mockAppConfig)
+    lazy val view = views.html.errors.sessionTimeout()(request, messages, mockConfig)
```

Comment:
As these vals are all implicit, you can omit them and just have `views.html.errors.sessionTimeout()`